sw/device/silicon_creator/lib/manifest.h

```diff
@@ -165,25 +166,55 @@ inline rom_error_t manifest_signed_region_get(
   return kErrorOk;
 }
 
-// FIXME: Remove this after adding the entry_point field.
-static_assert(sizeof(manifest_t) > 768 && sizeof(manifest_t) <= 1024,
-              "kEntryPointOffset must be updated");
+/**
+ * Gets the executable region of the image.
```

Comment:
I can see us wanting to load these values multiple times. For example, it may make sense to reload them when validating the ePMP configuration. Would it be better to validate all the fields up front rather than having the error checks in getters?

Reply:
Callers can hold onto the code region if they don't want to perform these checks again. We can definitely move these to a separate function, just thought it would be slightly safer to have these checks in the getters.

Reply:
Yeah, I think there are pros and cons either way. I'm happy to leave this as-is.